_dashboard/DotNetPerfMonitor/nuxt.config.ts

```diff
@@ -1,13 +1,23 @@
 // https://nuxt.com/docs/api/configuration/nuxt-config
 export default defineNuxtConfig({
   devtools: { enabled: true },
+  target: 'static',
+  app: {
+	baseURL: '/DotNetPerfMonitor/',
+  },
+  
   generate: {
     routes: ["/notifications", "/settings", "/documentation", "/regressions"],
   },
+  
   colorMode: {
     preference: "dark",
   },
   modules: ["@nuxthq/ui"],
 
   css: ["@/assets/css/global.css"],
+  
+  experimental: {
+	payloadExtraction: "false",
```

Comment:
I'm not sure if disabling the experimental payload extraction is necessary, but I did it anyway to get rid of a warning.